plugins/services/src/js/structs/Framework.js

```diff
@@ -18,6 +23,23 @@ module.exports = class Framework extends Application {
     this._spec = null;
   }
 
+  /**
+   * @override
+   */
+  getImages() {
+    const parsedImages = {};
+    const pkg = CosmosPackagesStore.getAvailablePackage(this.getPackageName());
```

Comment:
is there any other way than this? I know we did this before, but structs directly accessing stores is one of the issues we face when working with our codebase right now :( 

Maybe we can try to move this to the place where the struct is generated and inject the images there? 🤔 Without digging too deep I think `ServiceTree.js#constructor` is the "best" place for this - but first we need to reintegrate `ServiceUtil#createServiceFromResponse` back into the `ServiceTree` class… 😢

I'd vote for this refactoring 👍 
@nLight @orlandohohmeier opinions?

Reply:
Why not asking the whole team for opinions? :)

Reply:
@dcos/frontend-dev ^

Reply:
I second Daniel here.

Reply:
I thought that was related to the same _commit_, not _PR_. 🤔

Reply:
Also, this isnt a bug, its new code 🤔

Reply:
Okay, might be a misunderstanding on my side. I understood it as in not in the same PR/ticket.
It's new code fixing a bug. When we now go to change something in the structs I fear that we introduce new bugs with this refactoring.